packages/react-focus/src/components/FocusZone/FocusZone.test.tsx

```diff
@@ -78,10 +78,15 @@ describe('FocusZone', () => {
   isConformant({
     Component: FocusZone,
     displayName: 'FocusZone',
-    disabledTests: [
-      // Their is no existing top level FocusZone.ts file.
-      'has-top-level-file',
-    ],
+    // Disabled Tests:
+    //
+    // has-top-level-file
+    // Their is no existing top level FocusZone.ts file.
```

Comment:
```suggestion
    // There is no existing top level FocusZone.ts file.
```

Reply:
Also we should probably just create this file (it won't hurt anything and makes this package match the convention in case anyone expects that)